readme.md

```diff
@@ -133,5 +138,7 @@ Translations are in progress for German, Dutch, French, Hungarian, Polish, Spani
 - [GitHub](https://github.com/IntelOrca/OpenRCT2)
 - [Facebook](https://www.facebook.com/OpenRCT2)
 - [Automated Builds](https://openrct2.com/download)
+- [Secondary Site for Automated Builds](https://openrct.net/builds.php)  
```

Comment:
Should be "Automated builds" and "Secondary site for automated builds", respectively.